mainnet-assets.json

```diff
@@ -397,5 +397,15 @@
     "subId": "0x36470f078fb1e8d5922c44b01ffa7190258827f7a6de97edea790ca182503d72",
     "assetId": "0x4fc8ac9f101df07e2c2dec4a53c8c42c439bdbe5e36ea2d863a61ff60afafc30",
     "isVerified": true
+  },
+
+  {
+    "name": "Psycho Ducky",
+    "symbol": "PSYCHO",
+    "decimals": 9,
+    "contractId": "0xd2c72034938ce660e4cac0f0f24ff8a311efb1dbe03ccb8a8e3eaafa7ce47c7e",
+    "subId": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
Sub id for psycho is `0x0000000000000000000000000000000000000000000000000000000000000031` (taken from https://mainnet-explorer.fuel.network/assets/0x86fa05e9fef64f76fa61c03f5906c87a03cb9148120b6171910566173d36fc9e)